adapters/adair-uae.js

```diff
@@ -54,16 +55,16 @@ exports.fetchData = function (source, cb) {
       return cb({message: 'Failure to load data urls.'});
     }
     // Wrap everything in a try/catch in case something goes wrong
-    try {
+   // try {
```

Comment:
Was this commented out by mistake?

Reply:
Oops yes, I believe so, but it should be fixed now.